java/client/src/test/java/glide/api/RedisClientTest.java

```diff
@@ -267,4 +238,35 @@ public void info_with_empty_InfoOptions_returns_success() {
         assertEquals(testResponse, response);
         assertEquals(testPayload, payload);
     }
+
+    private static Stream<Arguments> getCommandsWithoutArgs() {
+        return Map.<RequestType, Function<RedisClient, CompletableFuture<String>>>of(
+                        ConfigRewrite, RedisClient::configRewrite,
+                        ConfigResetStat, RedisClient::configResetStat,
+                        Info, RedisClient::info,
+                        Ping, RedisClient::ping)
+                .entrySet()
+                .stream()
+                .map(e -> Arguments.of(e.getKey(), e.getValue()));
+    }
+
+    @SneakyThrows
+    @ParameterizedTest(name = "{0} returns success")
+    @MethodSource("getCommandsWithoutArgs")
+    public void no_arg_command_returns_success(
```

Comment:
I'm not convinced this is saving all that much effort.

Reply:
reverted